server.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-console */
+const http = require('http');
+const https = require('https');
+const hooks = require('./hooks.json');
+
+const PORT = 3000;
+
+function createCard(message) {
+  return JSON.stringify({
+    cards: [
+      {
+        header: {
+          title: 'Monitor Alert',
+        },
+        sections: [
+          {
+            widgets: [
+              {
+                keyValue: {
+                  content: `${message}`,
+                },
+              },
+            ],
+          },
+        ],
+      },
+    ],
+  });
+}
+
+function sendCard(domain, path, card, callback) {
+  const options = {
+    host: domain,
+    path,
+    method: 'POST',
+    headers: {
+      'Content-Type': 'applicdation/json',
+      'Content-Length': card.length,
+    },
+  };
+
+  const cb = (res) => {
+    let body = '';
+
+    res.on('error', (error) => {
+      console.log(error);
+      callback(error, null);
+    });
+
+    res.on('data', (chunk) => {
+      body += chunk;
+    });
+
+    res.on('end', () => {
+      callback(null, { res, body });
+    });
+  };
+
+  const req = https.request(options, cb);
+  req.write(card);
+  req.end();
+}
+
+http.createServer((req, res) => {
+  let body = [];
+
+  req.on('error', (err) => {
+    console.error(err);
+  }).on('data', (chunk) => {
+    body.push(chunk);
+  }).on('end', () => {
+    body = Buffer.concat(body).toString();
+    const bodyObject = JSON.parse(body);
+
+    const card = createCard(bodyObject.message);
+    const { domain } = hooks[bodyObject.team];
+    const { path } = hooks[bodyObject.team];
+
+    const callback = (err, data) => {
+      res.setHeader('Content-Type', 'application/json; charset=UTF-8');
+      res.writeHead(200);
+
+      if (err) {
+        res.end(JSON.stringify({
+          error: err,
+        }));
+      } else {
+        res.end(JSON.stringify({
```

Comment:
As we are using a curl to request to send this message, we could retrive something more simple as "OK":
```
      if (err) {
        res.end(JSON.stringify({
          error: err,
        }));
      } else {
        res.end("OK");
      }
```

Otherwise it retrives the whole body:
![image](https://user-images.githubusercontent.com/32379500/86536606-54dcaa00-bebf-11ea-9d0b-682cd38ba80f.png)